omod-common/src/test/java/org/openmrs/module/webservices/rest/web/api/impl/RestServiceImplTest.java

```diff
@@ -263,13 +246,11 @@ public void getResourceByName_shouldFailIfResourceForGivenNameCannotBeFound() th
 	@Test
 	public void getResourceByName_shouldFailIfResourceForGivenNameDoesNotSupportTheCurrentOpenmrsVersion() throws Exception {
 		
-		List<Class<? extends Resource>> resources = new ArrayList<Class<? extends Resource>>();
-		resources.add(AnimalResource_1_9.class);
-		resources.add(AnimalResource_1_11.class);
-		
+		List<Class<? extends Resource>> resources = new ArrayList<>();
+		resources.add(LegacyAnimalResource.class);
```

Comment:
Instead of using a fake resources, isn't better to use an existing resource like CohortResource1_8?